client/me/concierge/calendar-step.js

```diff
@@ -104,6 +106,7 @@ class CalendarStep extends Component {
 						isDefaultLocale={ isDefaultLocale( this.props.currentUserLocale ) }
 						key={ date }
 						onSubmit={ this.onSubmit }
+						signupForm={ this.props.signupForm }
```

Comment:
It doesn't seem like `CalendarCard` needs anything from `signupForm` other than `timezone` so can we just pass in the timezone instead?
```
signupForm={ this.props.signupForm.timezone }
```

Doing that will also simplify the required prop type definition

Reply:
Right, updated.